yggdrasil/values.yaml

```
@@ -4,13 +4,13 @@
 # ingressDomain: test.domain.com 
 
 # This values is if you want to use paths instead of subdomains.
-host: testHost.com
+host: ingress.lynderup.net
```

```suggestion
host: example.com
```
We should use example.com on main, as this value should be set on a cluster basis